docs/troubleshooting.md

```diff
@@ -57,7 +72,9 @@ Replace `contour cli lds` with `contour cli rds` for RDS, `contour cli cds` for
 
 ## Can't make kube-lego work with Contour
 
-If you use [kube-lego][0] for Let's Encrypt SSL certificates, kube-lego appears to set the ingress class on the ingress record it uses for the acme-01 challenge to `nginx`.
+[Cert-manager][8] is preffered over [kube-lego][0] for using Let's Encrypt SSL certificates with Contour.
+
+If you must use kube-lego for Let's Encrypt SSL certificates, kube-lego appears to set the ingress class on the ingress record it uses for the acme-01 challenge to `nginx`.
```

Comment:
I think we should just remove all kube-lego references and show how to use cert-manager.

Reply:
That works for me! How to use cert-manager belongs outside of the troubleshooting docs, so I won't include that in this PR.